kodilla-good-patterns/src/main/java/com/kodilla/goodPatterns/Aviation/Application.java

```diff
@@ -0,0 +1,30 @@
+package com.kodilla.goodPatterns.Aviation;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Application {
+
+    Airport airport1 = new Airport("Jasionka");
+    Airport airport2 = new Airport("Balice");
+    Airport airport3 = new Airport("Modlin");
+
+    Flight flight1 = new Flight(airport1, airport2);
+    Flight flight2 = new Flight(airport1, airport3);
+    Flight flight3 = new Flight(airport2, airport3);
+
+    Timetable timetable = new Timetable();
+    timetable.addFlight(Flight);
+
```

Comment:
Prawdopodobnie chodzilo Ci o cos takiego
```
    public static void main(String[] args) {
        Airport airport1 = new Airport("Jasionka");
        Airport airport2 = new Airport("Balice");
        Airport airport3 = new Airport("Modlin");

        Flight flight1 = new Flight(airport1, airport2);
        Flight flight2 = new Flight(airport1, airport3);
        Flight flight3 = new Flight(airport2, airport3);

        Timetable timetable = new Timetable();
        timetable.addFlight(flight1);
        timetable.addFlight(flight2);
        timetable.addFlight(flight3);
        
    }

Reply:
Właśnie o to chodziło i tak samiutko robiłem i nie mogło mi załapać metody addFlight dla obiektu timetable. 
Dzis jak odpaliłem i zrobiłem tak samo to załapało , a wczoraj to własnie nie i juz zbaraniałem bo to prosta sprawa i nie wiedziałem co jest grane. 
Dzieki